docs/content/en/docs-dev/user-guide/managing-application/customizing-deployment/script-run.md

```diff
@@ -8,12 +8,11 @@ description: >
 
 `SCRIPT_RUN` stage is one stage in the pipeline and you can execute any commands.
 
-> Note: This feature is at the alpha status.
+> Note: This feature is at the alpha status and only for the application kind of KubernetesApp.
```

Comment:
```suggestion
> Note: This feature is at the alpha status and currently only for the application kind of KubernetesApp.
```

Reply:
@khanhtc1202 
Thank you! Forgot to add it 🙏 
Added :) [3714161](https://github.com/pipe-cd/pipecd/pull/4774/commits/37141617d6af68563c38c3abc75e411184d2c839)